aptos-move/framework/aptos-framework/sources/configs/version.spec.move

```diff
@@ -37,7 +37,7 @@ spec aptos_framework::version {
         pragma verify_duration_estimate = 120;
         include transaction_fee::RequiresCollectedFeesPerValueLeqBlockAptosSupply;
         include staking_config::StakingRewardsConfigRequirement;
-        requires chain_status::is_operating();
+        requires chain_status::is_genesis();
```

Comment:
@rahxephon89 
[here](https://aptos.dev/move/prover/spec-lang/#requires-condition) it mentions `aborts_if` is preferred over `requires` for public functions?

Reply:
yes, unless this function is a public function that should never abort (such as block_prologue)